markdown/dev/reference/api/stack/asProps/en.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Stack.asProps()
+---
+
+The `Stack.asProps()` method returns a stack object suitable for renderprops.
+
+
+## Stack.asProps() signature
+
+```js
+Object stack.asProps()
+```
```

Comment:
I think this wants a `Returned Object` section, like in [Patter.getRenderProps()](https://github.com/freesewing/freesewing/blob/develop/markdown/dev/reference/api/pattern/getrenderprops/en.md#patterngetrenderprops-returned-object)

The `Stack.asProps()` method returns an object with 
the following properties:

| Property | Description |
| --------:| ----------- |
| `attributes` | An [`Attributes`](/reference/api/attributes) instance holding the stack's attributes |
| `bottomRight` | A [`Point`](/reference/api/point) that is the bottom right of the stack's bounding box |
| `height` | Height of the bounding box of the stack in `mm` |
| `name` | The name of the stack |
| `parts` | An `Array` of the `Part`s in the stack |
| `topLeft` | A [`Point`](/reference/api/point) that is the top left of the stack's bounding box |
| `width` | Width of the bounding box of the stack in `mm` |

Reply:
Added.